### PR TITLE
create third endpoint to add flower to database related #9

### PR DESCRIPTION
**Pull request by Nadeen123**

### CODE REVIEW

File: src/controllers/addflowerController.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const addFlower = require('../database/queries/addflower')
+exports.addone = (request,response)=>{
+
+    
+const input=request.body
+
+
+    addFlower(input.name,input. arabicname,input.imgUrl,input.description,(err,res)=>{
```
Comment (shahenazmonia) [inline src/controllers/addflowerController.js]: use destructuring `const {name,arabicname,imgUrl,description} =req.body; ` then pass the varaibles to `addFlower` function

File: src/database/queries/addflower.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const dbconnection = require("../db_connection");
+const addData = (name,arabicname,img,descrirtion,cb) => {
+    console.log(name,arabicname,img,descrirtion)
```
Comment (shahenazmonia) [inline src/database/queries/addflower.js]: delete the console log

Comment (Nadeen123) [reply src/database/queries/addflower.js]: resolve comment

File: src/controllers/addflowerController.js (hunk 1 above)
Comment (Nadeen123) [reply src/controllers/addflowerController.js]: resolve comment

**Pull request merged**